### ISSUE TITLE
Not able to read snappy compressed Parquet

### ISSUE THREAD

**Issue by dipeshkd** [slingdata-io/sling-cli#32]
Hi,

I am not able to read a compressed Parquet file.

I tried with following methods with errors:
- pandas
```
ArrowInvalid: Could not open Parquet input source '<Buffer>': Parquet file size is 51961168 bytes, smaller than the size reported by footer's (143101950bytes)
```

- Redshift Spectrum
```
Invalid metadata size in file footer: 143101950 bytes. File size: 51961168 bytes.
```
This could be related to https://github.com/slingdata-io/sling-cli/issues/31
Let me know if you need any additional information.

**Comment by flarco** [comment]
Thanks, do you think any particular conditions are causing the parquet file to be unreadable?
It seems that you were able to read other parquet files right? What was different? The default compression is actually Snappy.

**Comment by dipeshkd** [comment]
Yes I am able to read the parquet files without stating compression. I issue is when I try to read when explicitly selecting snappy.

The thing is the file size is different when I explicitly select snappy. Here are the target options:
- without selecting snappy `{file_max_rows: 10000, format: parquet)`
![image](https://github.com/slingdata-io/sling-cli/assets/9581711/334d6130-b1ff-4625-a5c9-fea930afe7e5)
- with snappy compression `{file_max_rows: 10000, format: parquet, compression: snappy}`
![image](https://github.com/slingdata-io/sling-cli/assets/9581711/7f6e96b1-5d69-4728-9007-d05a706b6c61)

**Comment by flarco** [comment]
@dipeshkd just released a new version (`1.0.47`). Could you try again?
